packages/celer-web-app/src/ui/frames/Home.tsx

```diff
@@ -1,40 +1,77 @@
 import { useState } from "react";
-import { LocalStorageWrapper } from "data/storage";
+import { View, Text } from "react-native";
+import { HomePageColors, Sizes, Fonts } from "ui/styles";
+import { LocalStorageWrapper, loadRecentPages } from "data/storage";
 import { EmptyObject } from "data/util";
 
+// Function that automatically populates the Recent Pages table
+function populateRecentPages() {
+	const recentPages = loadRecentPages();
```

Comment:
Still need to fix this - it's a performance issue if you load from localstorage on every re-render. I think in this case you only need to load it once? Since the list won't change if user stays on home page